### PR TITLE
Password change

### PR DESCRIPTION
**Pull request by AbhijitVyas**
Hi,

please check the pull request. 

Thanks,
Abhijit

### CODE REVIEW

**Comment by daniel86** [comment]
could you do the request against the *neem-hub* branch, please?

**Comment by AbhijitVyas** [comment]
sure. sorry for so many changes on pull request, I was testing it.

File: webrob/pages/main.py [hunk 1]
```diff
@@ -64,7 +71,31 @@ def render_main():
...
+@app.route('/change_password_post', methods=["POST"])
+@login_required
+def render_change_password_post():
+    form = PasswordForm()
+    if "Update" in request.form:
+        user = current_user
+        user.password = app.user_manager.hash_password(form.password.data)
+        db.session.add(user)
+        db.session.commit()
+        os.write(2, b'  Password has been updated! \n')
```
Comment (daniel86) [inline webrob/pages/main.py]: any reason not to use flask logging interface `app.logger.info`?

**Review by daniel86 — CHANGES_REQUESTED** [review]

File: webrob/pages/main.py (hunk 1 above)
Comment (AbhijitVyas) [reply webrob/pages/main.py]: nope, I just copied it from one of the previous classes. Or probably online. :)

Comment (AbhijitVyas) [reply webrob/pages/main.py]: I have changed the logger to more standard app.logger.info

**Review by daniel86 — APPROVED** [review]

**Review by daniel86 — APPROVED** [review]
looks good

**Pull request merged**